chpl/chpl-service/src/main/java/gov/healthit/chpl/Util.java

```diff
@@ -29,5 +31,27 @@ public static String md5(String input) {
 	    }
         return md5;
     }
+    

```

Comment:
Are the two methods below used anywhere? I don't see them used throughout this PR. I think they can be removed.